code/datums/diseases/advance/advance.dm

```diff
@@ -197,8 +198,6 @@ var/list/advance_cures = 	list(
 		properties["transmittable"] += S.transmittable
 		properties["severity"] = max(properties["severity"], S.level) // severity is based on the highest level symptom
 
-	return properties

```

Comment:
Shouldn't this cause issues? Specifically you're defining a local at line 191 with the same line as an instance var (on line 41). It is unclear to me which var would be modified.

I assume you want to delete line 191 here.